corehq/apps/data_cleaning/tests/test_session.py

```diff
@@ -124,3 +124,28 @@ def test_add_column_filters(self):
         for index, prop_id in enumerate(['watered_on', 'name', 'num_leaves', 'pot_type', 'height_cm']):
             self.assertEqual(column_filters[index].prop_id, prop_id)
             self.assertEqual(column_filters[index].index, index)
+
+    def test_reorder_column_filters(self):
```

Comment:
nit: I think it'd be slightly nicer to see this test split in two - one to test the wrong number of columns, and one to test the reordering as expected.

Reply:
fair point, i was being a little quick here. updated in https://github.com/dimagi/commcare-hq/pull/35846/commits/234198e6edd1e2ae34c1579af8b7849590a84782